server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/method/PushSubscriptionSetCreatePerformer.scala

```diff
@@ -87,3 +95,20 @@ class PushSubscriptionSetCreatePerformer @Inject()(pushSubscriptionRepository: P
       case (path, _) => SetError.invalidArguments(SetErrorDescription(s"Unknown error on property '$path'"))
     }
 }
+
+object PushSubscriptionSetCreateProcessor {
+  val PUSH_VERIFICATION_TO_PUSH_SERVER_TTL_DEFAULT: PushTTL = PushTTL.validate(15).toOption.get
```

Comment:
Why a TTL so low? Can't we keep the default?

Reply:
your mean is `PushTTL.MAX`?